src/vs/editor/browser/controller/textAreaInput.ts

```diff
@@ -622,11 +640,12 @@ export class TextAreaInput extends Disposable {
 	}
 
 	public writeScreenReaderContent(reason: string): void {
```

Comment:
should this be renamed to something inclusive of IMEs?

Reply:
Do you mean because it doesn't write when doing a composition? I don't think that's needed